src/main/java/core/basesyntax/Type.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Type {
```

Comment:
not informative name. Are you sure you need this enum? You can randomize figure just off the integers